Mobile/ShopLens/Droid/CameraActivity.cs

```diff
@@ -22,13 +22,17 @@ public class CameraActivity : Activity
     {
         Button BtnTakeImg;
         ImageView ImgView;
+        Button BtnPickImg;
+        public static readonly int PickImageId = 1000;
 
         public File productPhoto;
         public File _dir;
 
         public const int REQUEST_IMAGE = 102;
         public const string FILE_PROVIDER_NAME = ".shoplens.fileprovider";
 
+        public static int PICK_IMAGE = 1;
```

Comment:
Why is this variable not used anywhere? Was this supposed to be PickImageId?

Reply:
Maybe we should have a convention on activity Id number naming? For example, all activities that have something do to with photos start with two digits 1 and 0 or something like that?

Reply:
Or maybe all implicit activities also start with 2, explicit ones never start with 2?

Reply:
Yeah, we need to figure out what to do about those ids, maybe a separate class for the constants would be a good way

Reply:
This pick_image was meant for picked images count but since we don't need that it can be deleted